modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcComplexQuerySelfTest.java

```diff
@@ -209,21 +208,25 @@ public void testWrongArgumentType() throws Exception {
         }
 
         // Check non-indexed field.
-        assertThrows(SQLException.class, () -> {
-            try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Org where name = 2")) {
-                assertFalse(rs.next());
-            }
-        });
+        JdbcTestUtils.assertThrowsSqlException(
```

Comment:
Maybe we can use `import static` instead of `JdbcTestUtils.assertThrowsSqlException`, because most of the code uses `import static JdbcTestUtils.assertThrowsSqlException`.

Reply:
I don't consciously use it, due t have understanding that we expect JDBC exception.